api/Document.json

```diff
@@ -8846,7 +8922,7 @@
               "version_added": true
             },
             "opera_android": {
-              "version_added": true
+              "version_added": "14"
```

Comment:
This is a peculiar edit. chrome_android is true and wasn't changed, plus opera above was not mirrored. Did you intend to make this change?